src/app/features/auth/auth.module.ts

```diff
@@ -0,0 +1,37 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { Routes, RouterModule } from '@angular/router';
+
+// components
+import * as fromComponents from './components';
+
+// containers
+import * as fromContainers from './containers';
+
+const routes: Routes = [
```

Comment:
consider to move this routes into separate `child-routing.module`, each feature module should have their own separate `routing`, check in the docs how to do it [https://angular.io/guide/router#hero-feature-route-configuration](url)